WordPress/Resources/es.lproj/Localizable.strings

```diff
@@ -1503,6 +1503,9 @@ translators: Block name. %s: The localized block name */
 /* Title displayed for selection of custom app icons that have colorful backgrounds. */
 "Colorful backgrounds" = "Fondos coloridos";
 
+/* translators: %d: column index. */
+"Column %d" = "Columna %d";
```

Comment:
@startuptester as you can see here, the Spanish version has been translated, might be good to test against.